### ISSUE TITLE
Lab 1 - Exercises for Javascript 

### ISSUE THREAD

**Issue by davyie** [davyie/SDA-js_labs#11]
It might be good to include some exercises where the students write code in Javascript before jumping on the Calculator. It can be overwhelming if the only Javascript exercise is Calculator in Lab 1.

**Comment by kwabe007** [comment]
Agreed, I'll try to cook up some kind of js introduction hello-world-like thing!
